projects/plugins/jetpack/modules/search/instant-search/components/search-filters.scss

```diff
@@ -47,6 +47,8 @@
 		cursor: pointer;
 		width: 14px;
 		height: 14px;
+		top: initial;
```

Comment:
Not supported in IE11, but I also don't think we're striving for visual perfection in that particular browser.

https://caniuse.com/css-initial-value

Reply:
I'll go ahead and set them to `0` instead.